include/z64audio.h

```diff
@@ -0,0 +1,1186 @@
+    /* 0x10 */ SoundFontSound* soundEffects;
+} SoundFont; // size = 0x14
+
+typedef struct {
+    /* 0x00 */ u8* pc;
+    /* 0x04 */ u8* stack[4];
+    /* 0x14 */ u8 remLoopIters[4];
+    /* 0x18 */ u8 depth;
+    /* 0x19 */ s8 value;
+} SeqScriptState; // size = 0x1C
+
+// Also known as a Group, according to debug strings.
+typedef struct {
+    /* 0x000 */ u8 enabled : 1;
+    /* 0x000 */ u8 finished : 1;
+    /* 0x000 */ u8 muted : 1;
+    /* 0x000 */ u8 seqDmaInProgress : 1;
+    /* 0x000 */ u8 fontDmaInProgress : 1;
+    /* 0x000 */ u8 recalculateVolume : 1;
+    /* 0x000 */ u8 stopScript : 1;
+    /* 0x000 */ u8 unk_0b1 : 1;
+    /* 0x001 */ u8 state;
+    /* 0x002 */ u8 noteAllocPolicy;
+    /* 0x003 */ u8 muteBehavior;
+    /* 0x004 */ u8 seqId;
+    /* 0x005 */ u8 defaultFont;
+    /* 0x006 */ u8 unk_06[1];
+    /* 0x007 */ s8 playerIdx;
+    /* 0x008 */ u16 tempo; // tatums per minute
+    /* 0x00A */ u16 tempoAcc;
+    /* 0x00C */ s16 unk_0C;
+    /* 0x00E */ s16 transposition;
+    /* 0x010 */ u16 delay;
+    /* 0x012 */ u16 fadeTimer;
+    /* 0x014 */ u16 fadeTimerUnkEu;
+    /* 0x016 */ u16 unk_16; // New to MM
+    union {
+        /* 0x018 */ u8* seqData;
+        /* 0x018 */ u16* seqData16;
+    };
+    /* 0x01C */ f32 fadeVolume;
+    /* 0x020 */ f32 fadeVelocity;
+    /* 0x024 */ f32 volume;
+    /* 0x028 */ f32 muteVolumeScale;
+    /* 0x02C */ f32 fadeVolumeScale;
+    /* 0x030 */ f32 appliedFadeVolume;
+    /* 0x034 */ f32 unk_34;
+    /* 0x038 */ struct SequenceChannel* channels[16];
+    /* 0x078 */ SeqScriptState scriptState;
+    /* 0x094 */ u8* shortNoteVelocityTable;
+    /* 0x098 */ u8* shortNoteGateTimeTable;
+    /* 0x09C */ NotePool notePool;
+    /* 0x0DC */ s32 skipTicks;
+    /* 0x0E0 */ u32 scriptCounter;
+    /* 0x0E4 */ char unk_E4[0x74]; // unused struct members for sequence/sound font dma management, according to sm64 decomp
+    /* 0x158 */ s8 soundScriptIO[8];
+} SequencePlayer; // size = 0x160
+
+typedef struct {
+    /* 0x0 */ u8 releaseRate;
+    /* 0x1 */ u8 sustain;
+    /* 0x4 */ AdsrEnvelope* envelope;
+} AdsrSettings; // size = 0x8
+
+typedef struct {
+    union {
+        struct {
+            /* 0x00 */ u8 unk_0b80 : 1;
+            /* 0x00 */ u8 hang : 1;
+            /* 0x00 */ u8 decay : 1;
+            /* 0x00 */ u8 release : 1;
+            /* 0x00 */ u8 state : 4;
+        } s;
+        /* 0x00 */ u8 asByte;
+    } action;
+    /* 0x01 */ u8 envIndex;
+    /* 0x02 */ s16 delay;
+    /* 0x04 */ f32 sustain;
+    /* 0x08 */ f32 velocity;
+    /* 0x0C */ f32 fadeOutVel;
+    /* 0x10 */ f32 current;
+    /* 0x14 */ f32 target;
+    /* 0x18 */ char unk_18[4];
+    /* 0x1C */ AdsrEnvelope* envelope;
+} AdsrState; // size = 0x20
+
+typedef struct {
+    /* 0x0 */ u8 unused : 2;
+    /* 0x0 */ u8 bit2 : 2;
+    /* 0x0 */ u8 strongRight : 1;
+    /* 0x0 */ u8 strongLeft : 1;
+    /* 0x0 */ u8 stereoHeadsetEffects : 1;
+    /* 0x0 */ u8 usesHeadsetPanEffects : 1;
+} StereoData; // size = 0x1
```

Comment:
```suggestion
typedef struct {
    /* 0x0 */ u8 unused : 2;
    /* 0x1 */ u8 bit2 : 2;
    /* 0x2 */ u8 strongRight : 1;
    /* 0x3 */ u8 strongLeft : 1;
    /* 0x4 */ u8 stereoHeadsetEffects : 1;
    /* 0x5 */ u8 usesHeadsetPanEffects : 1;
} StereoData; // size = 0x6
```
Can 6 u8s be a size of `0x1` or is it supposed to be this?

Reply:
Makes sense, thanks for the explanation!